module/zfs/vdev_label.c

```diff
@@ -208,6 +208,40 @@ vdev_label_write(zio_t *zio, vdev_t *vd, int l, abd_t *buf, uint64_t offset,
 	    ZIO_PRIORITY_SYNC_WRITE, flags, B_TRUE));
 }
 
+void
+vdev_mmp_read(zio_t *zio, vdev_t *vd, int l, int n, abd_t *buf,
```

Comment:
(style) having variable names more descriptive than "l" and "n" would make it more clear what these values actually are.  For example, `vdev_label_num` and `mmp_uberblock_idx` or similar would be helpful.